### PR TITLE
autogenerate image samples in the docs

### PR DESCRIPTION
**Pull request by edgarriba**
### Description

Generates sample image for the augmentations to be added in the docs.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by shijianjian — APPROVED** [review]

**Comment by edgarriba** [comment]
@shijianjian @Borda seems that the paths of the images are not enough in the readthedocs server. In local was fine to me. Any ideas ?

**Comment by shijianjian** [comment]
Should be replacing the ~~~../images/image.png~~~ path to the absolute one as ~~~https://kornia.readthedocs.com/images/image.png~~~.

Haven't checked the real absolute path, but should be an easy fix.

**Comment by edgarriba** [comment]
I'll fix later today

**Comment by Borda** [comment]
Also, debug it directly with building in readthedocs - temp enable the PR branch

File: docs/generate_example_images.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+for aug_name, args in augmentations_list.items():
+    # dynamically create the class instance
+    cls = getattr(mod, aug_name)
+    aug = cls(*args, p=1.0)
+    # apply the augmentaiton to the image and concat
+    out = aug(img_t)
+    out = torch.cat([img_t, out], dim=-1)
+    # save the output image
+    out_np = K.utils.tensor_to_image((out * 255.0).byte())
+    cv2.imwrite(str(OUTPUT_PATH / f"{aug_name}.jpg"), out_np)
```
Comment (shijianjian) [inline docs/generate_example_images.py:68]: I think the generated images are not hosted in the website. The URL is correct.

File: docs/generate_example_images.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import numpy as np
+import requests
+import torch
+
+import kornia as K
+
+mod = importlib.import_module("kornia.augmentation")
+
+BASE_IMAGE_URL: str = "https://raw.githubusercontent.com/kornia/data/main/panda.jpg"
+OUTPUT_PATH = Path(__file__).absolute().parent / "source/_static"
```
Comment (shijianjian) [inline docs/generate_example_images.py:14]: As a hotfix, a simple copy and paste should work:
```python
OUTPUT_PATH = Path(__file__).absolute().parent / "build/html/_static"
os.makedirs(OUTPUT_PATH, exist_ok=True)
```

Comment (edgarriba) [reply docs/generate_example_images.py:14]: @shijianjian didn't seem to work
